config/environments/production.rb

```diff
@@ -80,14 +80,34 @@
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  if true # ENV["RAILS_LOG_TO_STDOUT"].present?
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
-    config.logger    = ActiveSupport::Logger.new(STDOUT)
+    config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
+  config.logger           = ActiveSupport::Logger.new(STDOUT)
```

Comment:
@kuisathaverat Added this after the `if` block. No need to update the `Dockerfile` with the ENV var.

Reply:
I am not sure, Did you test it and the output is pure JSON?